cocos/input/input.ts

```diff
@@ -229,6 +229,41 @@ export class Input {
         }
         this._eventTarget.off(eventType, callback, target);
     }
+
+    /**
+     * @en
+     * Get touch object by touch ID.
+     * @zh
+     * 通过 touch ID 获取 touch对象
+     * @param touchID
+     * @returns
+     */
+    public getTouch (touchID: number, x: number, y: number): Touch | undefined {
+        return touchManager.getTouch(touchID, x, y);
+    }
```

Comment:
```suggestion
    /**
     * @en
     * Get touch object by touch ID.
     * @zh
     * 通过 touch ID 获取 touch 对象。
     * @param touchID
     * @returns The Touch object with specified touch id if exists.
     */
    public getTouch (touchID: number): Touch | undefined {
        return touchManager.getTouch(touchID, x, y);
    }
```

这里可能是 `touchManager. getTouch` 的接口命名误导你了，`touchManager.getTouch` 应该叫 `touchManager.getOrCreateTouch` 比较贴切，而开发者需要的应该是 `input.getTouch(touchID: number): Touch | undefined`

这里 input.getTouch 传入 x y 坐标并没有意义，也不符合接口的文档规范

Reply:
接口实现上可能不能直接调用 `touchManager.getTouch((touchID, x, y))`
估计需要 `touchManager._touchMap.get(touchID)`

`_touchMap` 属性应该需要 public 出来